Program.cs

```diff
@@ -18,17 +18,20 @@ static void Main()
             return;
         }
 
-        // Console.WriteLine("Enter the operation (+, -, *, /):");
-        // char operation = Console.ReadKey().KeyChar;
+        Console.WriteLine("Enter the operation (+, *):");
```

Comment:
https://github.com/coderabbitai generate case statements for division, subtraction, and exponents